resources/schema.py

```diff
@@ -328,6 +351,138 @@ def mutate_and_get_payload(cls, root, info, **kwargs):
         return DeleteBerthTypeMutation()
 
 
+class CreateHarborMutation(graphene.ClientIDMutation):
+    class Input:
+        area = HarborAreaInput(required=True)
+        municipality = graphene.String()
+        image_file = graphene.String()
+        availability_level = graphene.ID()
+        number_of_places = graphene.Int()
+        maximum_width = graphene.Int()
+        maximum_length = graphene.Int()
+        maximum_depth = graphene.Int()
```

Comment:
you can skip these, as they will be calculated automatically later, ref VEN-362 🙂

Reply:
Should we remove them right away or wait until 362 is developed? To ensure we have support them.

Reply:
well, if they work now in these mutations, then we can keep 'em I guess

Reply:
They do work. Let's keep them then for now and I'll add a comment to the ticket to remind me to remove them.